.swiftlint.yml

```diff
@@ -1,6 +1,6 @@
-line_length:
-  warning: 150
-  ignores_comments: true
 
 disabled_rules:
-  identifier_name
+- identifier_name
+- trailing_whitespace
+- line_length
```

Comment:
I do like the `trailing_whitespace` rule 🤔 Same with `line_length` but this is so trivial, let's leave it for now.